Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderDataSystem.cpp

```diff
@@ -33,28 +34,31 @@ namespace ScriptCanvasBuilder
     DataSystem::DataSystem()
     {
         AzToolsFramework::AssetSystemBus::Handler::BusConnect();
```

Comment:
The Builder Data system now tracks the state of building source files -> editor data (allowing the user to configure overrides from a graph when used on a component or in the widget) AND it keeps track of which runtime assets have been properly built and loaded. Clients interested in both either do not need to talk to asset systems or the AP to get the details. Usually, they will only care with the builder data system tells them. It also shares cached successful results of compilation so source assets aren't continually reloaded or processed.